src/SVGPathData.ts

```diff
@@ -85,6 +85,7 @@ export const COMMAND_ARG_COUNTS = {
     [SVGPathData.ARC]: 7,
 };
 
-export {encodeSVGPath} from "./SVGPathDataEncoder";
-export {SVGPathDataParser} from "./SVGPathDataParser";
-export {SVGPathDataTransformer} from "./SVGPathDataTransformer";
+export {encodeSVGPath} from "./SVGPathDataEncoder.js";
+export {SVGPathDataParser} from "./SVGPathDataParser.js";
+export {SVGPathDataTransformer} from "./SVGPathDataTransformer.js";
+export * from "./types.js";
```

Comment:
Put this here because [`canvg` wants it](https://github.com/canvg/canvg/blob/57afaaaf45e45fb79c7dff430d702e0777faa22c/src/PathParser.ts#L1-L13).